### ISSUE TITLE
Space between interpolated statements sometimes removed

### ISSUE THREAD

**Issue by Andrew-Marks-Trisept** [daniel-sc/ng-extract-i18n-merge#17]
When there are multiple interpolated statements in a row with a space in between, the space is sometimes removed.  This only seems to happen as the message gets more complex.  Consider the following examples:

**HTML**
```
<span i18n="@@myTestId">{{ myLabel }} {{ myNumber }}</span>
<span i18n="@@myTestId2">Some text {{ myLabel }} {{ myNumber }}</span>
```

**.ts**
```
myLabel: string = 'Thing';
myNumber: number = 1;
```

**xlf Output**
```
<unit id="myTestId">
  <segment state="final">
	<source>
	  <ph id="0" equiv="INTERPOLATION" disp="{{ myLabel }}"/>
	  <ph id="1" equiv="INTERPOLATION_1" disp="{{ myNumber }}"/>
	</source>
	<target>
	  <ph id="0" equiv="INTERPOLATION" disp="{{ myLabel }}"/>
	  <ph id="1" equiv="INTERPOLATION_1" disp="{{ myNumber }}"/>
	</target>
  </segment>
</unit>
<unit id="myTestId2">
  <segment state="final">
	<source>Some text <ph id="0" equiv="INTERPOLATION" disp="{{ myLabel }}"/><ph id="1" equiv="INTERPOLATION_1" disp="{{ myNumber }}"/></source>
	<target>Some text <ph id="0" equiv="INTERPOLATION" disp="{{ myLabel }}"/><ph id="1" equiv="INTERPOLATION_1" disp="{{ myNumber }}"/></target>
  </segment>
</unit>
```

The first example works and outputs `Thing 1` -- I think by virtue of the line break in between the `<ph>` elements?

The second example outputs `Some text Thing1` when it should be `Some text Thing 1`.

**Comment by Andrew-Marks-Trisept** [comment]
So I was thinking maybe this wasn't related to this library, because it's like that in the extracted source file.  I thought maybe something changed in Angular 13.

But then I went back and ran the default `ng extract-i18n` script like I used to and sure enough, the space was added properly.  Any idea what the difference would be?  I assumed your `extract-i18n-merge` just ran `extract-18n` in the background.

Maybe something with the alphabetizing of the nodes causes this?
